plugin/index.php

```diff
@@ -1,14 +1,18 @@
 <?php
-
 /**
  * Plugin Name: Gutenberg
  * Plugin URI: https://wordpress.github.io/gutenberg/
  * Description: Prototyping since 1440. Development plugin for the editor focus in core.
  * Version: 0.1
  */
-
+// Hook.
```

Comment:
My personal view is that a comment shouldn't exist to describe _what_ exists, but _why_ (more specifically in clarifying potentially ambiguous logic). In this case, "Hook" is already quite clear merely by the invoking `add_action`.